consensus/safety-rules/src/process.rs

```diff
@@ -49,20 +51,29 @@ struct ProcessData {
     server_addr: SocketAddr,
     storage: PersistentSafetyStorage,
     verify_vote_proposal_signature: bool,
+    // Timeout in Seconds for network operations
+    network_timeout: u64,
 }
 
 pub struct ProcessService {
     server_addr: SocketAddr,
+    network_timeout: u64,
 }
 
 impl ProcessService {
-    pub fn new(server_addr: SocketAddr) -> Self {
-        Self { server_addr }
+    pub fn new(server_addr: SocketAddr, network_timeout: u64) -> Self {
+        Self {
+            server_addr,
+            network_timeout,
+        }
     }
 }
 
 impl RemoteService for ProcessService {
     fn server_address(&self) -> SocketAddr {
         self.server_addr
     }
+    fn network_timeout(&self) -> u64 {
```

Comment:
I wish this was a lint..... new line.

Reply:
It ought to be if it's not.  (and it's not locally).

BTW, I observe two things right now:

1). `cargo xclippy` seems to have been removed?
2) `cargo xtest` seems to have been removed?
3). I can no longer run circles integration tests (which may explain not hitting the ought to exist lint rule,  if it does in fact exist)

Reply:
As of a couple days ago, these have been changed.

You should be able to do  `cargo xlint`, `cargo clippy` instead as well as `cargo test`, but you'll probably have to do `cargo test -p <package> --all-features` or enable the testing features other ways.

Clippy might not be quite as smart without the test features enabled.  Hopefully, these will be fixed in the future to work again :)